kde-apps/kate-lib/kate-lib-9999.ebuild

```diff
@@ -39,15 +39,12 @@ DEPEND="
 	telemetry? ( >=kde-frameworks/kuserfeedback-${KFMIN}:6 )
 "
 RDEPEND="${DEPEND}
-	!<kde-apps/kate-22.08.0:5
+	>=kde-apps/kate-common-${PV}
 "
 
 src_prepare() {
 	ecm_src_prepare
-
-	# delete colliding kate/kwrite translations
-	find po -type f -name "*po" -and ! -name 'kate.po' -delete || die
-	rm -rf po/*/docs || die
+	ecm_punt_po_install
```

Comment:
Sort of wonder if we should have `ecm_src_prepare` take a var for it but it's not a big win and it's just shifting the logic. But I generally like being declarative...

Anyway, don't worry about it for now.

Reply:
I thought about that as well and even spun it as far as considering separate `${PN}-i18n` and `${PN}-handbook` packages to be referred in aptly named meta vars (then we could trigger such functions by eclass as well), but then with a `-handbook` only package we can not have it as optional RDEPEND again...

Reply:
Gah. Yet another reason to bump it up my list..